src/components/Sidebar/BranchTransition.tsx

```diff
@@ -27,12 +80,38 @@ const BranchTransition: FC<BranchTransitionProps> = ({ pathway, transition, upda
         updatePathway={updatePathway}
         isTransition={true}
       />
+      {!useCriteriaSelected && (
+        <SidebarButton
+          buttonName="Use Criteria"
+          buttonIcon={<FontAwesomeIcon icon={faPlus} />}
+          buttonText="Add previously built or imported criteria logic to branch node."
+          onClick={handleUseCriteria}
+        />
+      )}
 
-      <SidebarButton
-        buttonName="Use Criteria"
-        buttonIcon={<FontAwesomeIcon icon={faPlus} />}
-        buttonText="Add previously built or imported criteria logic to branch node."
-      />
+      {useCriteriaSelected && (
+        <>
+          <DropDown
+            id="Criteria"
+            label="Criteria"
+            options={criteriaOptions}
+            onChange={selectCriteriaSource}
+            value={transition.condition?.cql || undefined}
```

Comment:
What happens here if `transition.condition.cql` is not in `criteriaOptions`?

Reply:
The dropdown doesn't display a value